subiquity/models/installpath.py

```diff
@@ -40,25 +41,48 @@ def paths(self):
 
     def update(self, results):
         if self.path == 'ubuntu':
-            self.packages = {}
-            self.debconf = {}
+            self.source = '/media/filesystem'
+            self.curtin = {}
         elif self.path == 'maas_region':
-            self.packages = {'packages': ['maas']}
-            self.debconf['debconf_selections'] = {
+            self.source = '/media/region'
+            self.curtin['debconf_selections'] = {
                 'maas-username': 'maas-region-controller maas/username string %s' % results['username'],
                 'maas-password': 'maas-region-controller maas/password password %s' % results['password'],
-                }
+            }
+            self.curtin['late_commands'] = {
+                '900-maas': ['rm', '-f', '/target/etc/maas/rackd.conf'],
+                '901-maas': ['rm', '-f', '/target/etc/maas/region.conf'],
+                '902-maas': ['curtin', 'in-target', '--', 'maas-rack', 'config', '--init'],
```

Comment:
please add some code comments here to explain why these commands are in the order they are.  E.g. why do you init the config before you reconfigure the package?  Shouldn't we expect dpkg-reconfigure to take care of everything and make the separate 'maas-rack' invocation unnecessary?  Why are two of these commands run through curtin, but others done manually with chroot /target?

Reply:
`curtin in-target` does a lot of things, specifically, it does mount/unmount `/dev` and `/log`, and it unshares a pid namespace... which would break talking to the postgresql database. Hence only some commands are run via curtin in-target....

`--init` is not run on reconfigure, only on configure. And it initializes uuid, which i'm told is actually unused anymore... but little did I know... hence this matches everything what a udeb/deb based install will do.